src/wasm/wasm.cpp

```diff
@@ -1395,6 +1436,49 @@ void Resume::finalize(Module* wasm) {
       this->contType.getContinuation().type.getSignature();
     type = contSig.results;
   }
+
+  populateResumeSentTypes(this, wasm);
```

Comment:
Hahaha, sorry for the back and forth. I think we're talking past each other here. I still think it doesn't make sense to populate the sent types in `finalize`. Instead, we should for the sent types to be populated from the start by whoever creates a `Resume` or `ResumeThrow` expression. In other words, `sentTypes` should be added as a required parameter to `Builder::makeResume` and `Builder::makeResumeThrow`.

Reply:
Yes, this was a mistake. I forgot to remove this code again after I reverted the commit. Please see commit https://github.com/WebAssembly/binaryen/pull/7041/commits/c8d345a900151fffe69df37d99a27b7076de3c26.